engine/src/rendering/vulkan/renderer.cc

```diff
@@ -0,0 +1,39 @@
+#include <spear/rendering/vulkan/renderer.hh>
+#include <iostream>
+#include <GL/glew.h>
+
+namespace spear::rendering::vulkan
+{
+    // FIXME: Not really a fan of this weird indentation scheme. Sorry but I am not going to see the trouble and change the indentation unless reasoned
+    //  why namespace {}-brackets are an exception.
+    //  T: soy dev.
```

Comment:
Please remove this. Since we don't enforce a specific format style yet, for now I suggest following @viirret's indentation guidelines and adjust your text editor settings accordingly.

Reply:
:+1: